pai-management/bootstrap/prometheus/yarn-exporter-ds.yaml.template

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Is yarn_exporter expected to run in every nodes or just one? If just one, deployment would be better?

Also, I recommend using [sidecar pattern](https://kubernetes.io/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume/). As it will not incur network overhead between two nodes, and easy deployment, since exporter and main program are using same pod.

Reply:
Just one. 
yarn-exporter is a single pod, as we discussed before, it's better to independent with node-exporter?
I will change it to deployment.

Reply:
Have changed to sidecar